package.json

```diff
@@ -16,5 +20,10 @@
     "./package.json": "./package.json",
     "./processRequest.mjs": "./processRequest.mjs",
     "./Upload.mjs": "./Upload.mjs"
+  },
```

Comment:
agora que está usando o TS já dá para fixar a versão das libs (tirar o ˆ) só para garantir que quem for clonar o repo (ou num processo de deploy) instale as libs na mesma versão em que o projeto foi desenvolvido e testado localmente o/